content/terraform-enterprise/1.0.x/docs/enterprise/deploy/reference/configuration.mdx

```diff
@@ -615,8 +615,12 @@ Required when `TFE_OBJECT_STORAGE_TYPE` is `s3`.
 
 #### `TFE_OBJECT_STORAGE_S3_ENDPOINT`
 
-S3 endpoint. Useful when using a private S3 endpoint. Leave blank to use the
-default AWS S3 endpoint. Defaults to `""`.
+S3 endpoint. Useful when using a private S3 endpoint.
+
+Example: `TFE_OBJECT_STORAGE_S3_ENDPOINT: https://s3.example.com`
```

Comment:
Sorry I can't add this as a suggestion because the markdown breaks, but could you tweak this to explain the example a bit like so:


The following example sets your S3 endpoint to a private S3 endpoint named `https://s3.example.com` :

```shell-session
TFE_OBJECT_STORAGE_S3_ENDPOINT: https://s3.example.com
```